book/website/collaboration/inclusive-events/inclusive-events-schedules.md

```diff
@@ -0,0 +1,286 @@
+
+```{note}
+Mozilla has developed a framework called [ADIDS](https://docs.google.com/presentation/d/1k6Sft7YH2QrDzqzE6TZhociWVjXe7rLMDjAJ734qNKA/edit#slide=id.g1cd292f458d_0_19) that can help event organizers plan sessions that actively engage and involve a diverse audience.
+You can read more about ADIDS on the [MozFest website](https://www.mozillafestival.org/en/facilitator-guide/designing-a-successful-session/), or by working through the framework for your own event.
+```
+(cl-inclusive-events-schedule-rapport)=
+### Building Rapport with Event Design
+The active involvement of people with lived experience is incredibly important because it enables the design and delivery of research and outputs that are informed by lived experiences, and better reflects the needs of the communities that stand to benefit the most from research interventions.
+
+You can encourage inclusive participation from people with lived experience by making use of inclusive and "unconventional" approaches in your event. 
+For example, consider brokering conversation between attendees in a "World Cafe". 
+
+World Cafes bring rotating tables (virtual or physical) of people with differing expertise together to engage in evolving rounds of dialogue on a specific topic. 
+The smaller grouping allows for more conversation between people and can be less daunting for people with lived experience to engage in. 
+You can read more about the World Cafe method on [The World Café Community Foundation's blog](https://theworldcafe.com/key-concepts-resources/world-cafe-method/). 
+
+(cl-inclusive-events-moderators-facilitators)=
+### The role of Moderators and Facilitators
+Facilitators and session moderators can help to foster a safe and supportive environment for people with lived experience to share their perspectives and insights. 
+Their role at an inclusive event is to provide safe and supported opportunities for active participation from attendees, act as a catalyst for conversation, and to help align discussion and any questions, with the goals of the session and event. 
+
+If you are able to involve facilitators or moderators at your event, consider how they can support the engagement of - and contributions from - people with lived experience. 
+This is especially important for events that focus on discussion or require active participation (such as the World Cafes we discussed in the previous section). 
+Many people with lived experience may not have a lot of experience, or had the opportunity to develop skills in public speaking or rhetoric and can therefore be at a greater risk of getting left out of conversation. 
+
+Just inviting people with lived experience to your event is not sufficient if they are not supported and actively welcomed to engage in your event's content. 
```

Comment:
I feel something should be added that the burden to participate should not be on those with lived experience, but should be carried by the facilitators which should also read up on neurodiversity and barriers to participation. This links in with the earlier comment I had on what facilitators can really do to help here - it is beyond facilitation of just neuro-typicals.. Sorry just rambling here and jotting this down for myself as well so that I can hopefully add this later.